src/organizations/Organizations.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useContext, useEffect } from "react";
+import { Table } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { AuthContext } from "../AuthContext";
+import { BASE_API } from "../config";
+import { SERVICE_UNAVAILABLE_ERROR } from "../messages";
+
+export default function Organizations() {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [organizations, setOrganizations] = useState([]);
+  const { access_token } = useContext(AuthContext);
+
+  const requestOrganizationsList = {
+    method: "GET",
+    headers: {
+      "Authorization": `Bearer ${access_token}`,
+      "Accept": "application/json",
+      "Content-Type": "application/json",
+    },
+  };
+
+  useEffect(() => {
+    fetch(`${BASE_API}/organizations`, requestOrganizationsList)
+      .then(async response => {
+        const data = await response.json();
+        if (response.ok) {
+          return setOrganizations(data);
+        }
+        setErrorMessage(data.message);
+      })
+      .catch(() =>
+        setErrorMessage(SERVICE_UNAVAILABLE_ERROR)
+      )
+
+  }, []);
+
+  return errorMessage ?
+    <div className="container-fluid" id="organizationsList">
+      <div className="top">
+        <h1>
+          {errorMessage}
+        </h1>
+      </div>
+    </div>
+    :
+    <>
+      <div className="container" id="organizationsList">
+        <div className="row mb-5">
+          <div className="col-lg-12 text-center">
+            <h1 className="mt-5">Organizations List</h1>
+          </div>
+        </div>
+        <Table striped bordered hover>
+          <thead>
+            <tr>
+              <th>Representative Name</th>
+              <th>Organization Name</th>
+              <th>Email</th>
+              <th>Website</th>
+              <th>Timezone</th>
+              <th>Phone</th>
+              <th>Total Programs</th>
+              <th>Available</th>
+              <th>In Progress</th>
+              <th>Completed</th>
+            </tr>
+          </thead>
+          <tbody>
+            {organizations.map((organization) => (
+              <tr key={organization.id}>
+              <td>{organization.representative_name}</td>
+                <td><Link to={{
+                  pathname: `/organizations/${encodeURI(organization.organization_name)}`,
```

Comment:
this needs to be /organization. that's what is mentioned in the Routes. I tried removing the s, I can see the detail view

Reply:
I see. Thanks for pointing that out, @meenakshi-dhanani . 👍